s3/src/main/scala/akka/stream/alpakka/s3/impl/S3Stream.scala

```diff
@@ -98,10 +98,14 @@ import akka.util.ByteString
   val MaxChunkSize: Int = 10 * 1024 * 1024 //in bytes
 
   // def because tokens can expire
-  def signingKey(implicit settings: S3Settings) = SigningKey(
-    settings.credentialsProvider,
-    CredentialScope(LocalDate.now(ZoneOffset.UTC), settings.s3RegionProvider.getRegion, "s3")
-  )
+  def signingKey(implicit settings: S3Settings) = {
+    val requestDate = ZonedDateTime.now(ZoneOffset.UTC)
+    SigningKey(
+      requestDate,
+      settings.credentialsProvider,
+      CredentialScope(requestDate.toLocalDate, settings.s3RegionProvider.getRegion, "s3")
```

Comment:
We are still seeing akka.stream.alpakka.s3.S3Exception: The authorization header is malformed; Invalid credential date. Date is not the same as X-Amz-Date. with alpakka 1.0.1.

I'm wondering, does going back to localDate in the CredentialScope screw things up? Because now requestDate and CredentialScope.date are going to have different dates, causing S3 to reject the request still.

Reply:
I do not see how the dates can be different.
From `toLocalDate` Javadoc: "This returns a LocalDate with the same year, month and day as this date-time."

Reply:
Okay I see I misunderstood and thought it would convert it to a date in the "local" timezone. It is working as expected. Thank you for clarifying!